R/fit_SE.R

```diff
@@ -15,6 +15,8 @@ fit_SE <- function(se,
 
```

Comment:
Maybe description for param could be added?
(description for `gDRutils::fit_curves` may help)

Reply:
Since it is a part of runDrugResponseProcessingPipeline, the parameters are described in runDrugResponseProcessingPipelineFxns. There is a separate task for reeavulation of fit_SE.combinations and I suggest reeavulate parameters descritipn for the whole process in that task.